customer-service/src/main/java/com/spaghettininjas/yaposs/repository/entity/Customer.java

```diff
@@ -0,0 +1,50 @@
+package com.spaghettininjas.yaposs.repository.entity;
+
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Entity
+@Data
+@Table(name = "customer")
+public class Customer {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String email;
+
+    private  String loyaltyLevel;
+
+    private String loyaltyPoints;
```

Comment:
Shouldn't this be an `int`?

Reply:
Yes, loyalty Points should be int

Reply:
Fixed according to [contractV2.yaml](https://github.com/domanteli0/PoSS/blob/main/contractV2.yaml)